scipy/optimize/tests/test__optimize.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
```

Comment:
These tests should go into `optimize/tests/test_optimize.py`, not in a separate file.

Reply:
I moved everything to the end of `optimize/tests/test_optimize.py`.